wave_utils/CMakeLists.txt

```diff
@@ -18,7 +20,10 @@ IF(BUILD_TESTING)
         tests/utils/data_test.cpp
         tests/utils/file_test.cpp
         tests/utils/math_test.cpp
-        tests/utils/time_test.cpp)
+        tests/utils/time_test.cpp
+        tests/utils/test_wave_spatial_utils.cpp
```

Comment:
This file contains an `int main()`. Is that not redundant since this test executable was already working?

Reply:
Removed `test_wave_spatial_utils.cpp`